src/main/java/org/symphonyoss/integration/webhook/github/parser/v2/GithubPullRequestMetadataParser.java

```diff
@@ -72,10 +81,27 @@ protected String getMetadataFile() {
   @Override
   protected void preProcessInputData(JsonNode input) {
     proccessIconURL(input);
+    processActionVerbs(input);
     processUser(input.path(SENDER_TAG));
     processUser(input.path(ASSIGNEE_TAG));
     processURL(input.path(PULL_REQUEST_TAG), HTML_URL_TAG);
     processURL(input.path(PULL_REQUEST_TAG).path(HEAD_TAG).path(REPO_TAG), HTML_URL_TAG);
     processURL(input.path(PULL_REQUEST_TAG).path(BASE_TAG).path(REPO_TAG), HTML_URL_TAG);
   }
+
+  /**
+   * Adjust some actions verbs to be more readable or grammar consistent.
+   * @param node JSON input payload
+   */
+  private void processActionVerbs(JsonNode node) {
+    String action = node.path(ACTION_TAG).textValue();
+    if (GITHUB_ACTION_SYNCHRONIZE.equals(action)) {
+      action = "synchronized";
+    } else if (GITHUB_ACTION_REVIEW_REQUESTED.equals(action)) {
+      action = "review requested";
+    } else if (GITHUB_ACTION_REVIEW_REQUEST_REMOVED.equals(action)) {
+      action = "review request removed";
+    }
+    ((ObjectNode) node).put(ACTION_TAG, action);
```

Comment:
According to the https://perzoinc.atlassian.net/browse/APP-981 synchronize event should be replaced by 'updated'.

I believe you might use a dynamic approach instead of a static approach using 'if' clause.

Something like that:

```
if (actionsLabel.containsKey(action)) {
    action = actionsLabel.get(actionName);
    ((ObjectNode) node).put(ACTION_TAG, action);
}
```

You should populate 'actionsLabel' in the constructor mapping the event against text to be displayed.